clang/include/clang/CIR/CIRLoweringHelpers.h

```diff
@@ -0,0 +1,43 @@
+//====- CIRLoweringHelpers.h - Lowering helper funcstions -----------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file declares helper functions for lowering from CIR to LLVM or MLIR.
+//
+//===----------------------------------------------------------------------===//
+#ifndef LLVM_CLANG_LIB_CIRLOWERINGHELPERS_H
+#define LLVM_CLANG_LIB_CIRLOWERINGHELPERS_H
```

Comment:
The macro name should follow the dir structure.

Reply:
Thanks for catching this!